Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -61,14 +61,16 @@ itkNiftiReadWriteDirectionTest(int ac, char * av[])
   // The sforms should be identical in both cases.
   if (LPSLabel_direction != LPSLabel_noqform_direction)
   {
+    std::cerr << "Error: sform direction matrices differ when qform is not present" << std::endl;
     return EXIT_FAILURE;
   }
   // Verify that the qform alone is not identical to the sform (due to lossy storage capacity)
   // This difference is small, but periodically causes numerical precision errors,
   // VERIFY THAT THE DIRECTIONS ARE ACTUALLY DIFFERENT, They are expected to be different due to lossy
-  // representaiton of the qform.
+  // representation of the qform.
   if (LPSLabel_direction == LPSLabel_nosform_direction)
   {
+    std::cerr << "Error: direction matrices from sform and qform are identical" << std::endl;
```

Comment:
In order to convey a more clear information about the error rationale, I would rephrase this as:
```
std::cerr << "Error: direction data must be different for qfrom- and sform-complete and qform-only versions of the same image due to lossy storage capacity." << std::endl;
```

@cookpa Do you think it is worthwhile adding some further information about the failure ? Something like, e.g.
```
std::cerr << "qfrom and sform-complete image direction: " << LPSLabel_direction << std::endl;
std::cerr << "qform-only image direction:" << LPSLabel_nosform_direction << std::endl;
```

Reply:
Addressed in 7f16366.